packages/rich-text/src/test/create.js

```diff
@@ -61,7 +61,7 @@ describe( 'create', () => {
 		formatName,
 		formatType,
 		html,
-		value,
+		value: expectedValue,
```

Comment:
I thought we could update it inside the definition,too :)

I can do it myself when working on e2e tests.

Reply:
Oh. It is used for other tests as well where `value` is the input.